readme.md

```diff
@@ -0,0 +1,23 @@
+# Laravel Media plugin
+
+## Introduction
+
+This plugin adds the ability for users to work with media files.
+
+## Installation
+
+1. Install the package using the following command: `composer require ronasit/laravel-media`
+2. Run `php artisan vendor:publish`
+3. For Laravel <= 5.5 add `RonasIT\Media\MediaServiceProvider::class` to config `app.providers` list.
+4. In `media.permitted_types` config, specify the allowed types of media files.
+5. In `media.classes.user_model` config, specify the model class that will have a relationship with the media class.
+
```

Comment:
```suggestion

## Integration with [LaravelSwagger](https://github.com/RonasIT/laravel-swagger)

This package includes OpenAPI documentation file. To include it to your project's documentation, you need to register it in the `auto-doc.additional_paths` config:

`vendor/ronasit/laravel-media/documentation.json`
```

Reply:
Done.

Reply:
nope, please check empty lines in the suggestion

Reply:
Fixed.